src/xg.cpp

```diff
@@ -1047,8 +982,10 @@ void XG::build(vector<pair<id_t, string> >& node_label,
     cerr << "total size [MB] = " << (
         size_in_mega_bytes(s_iv)
         //+ size_in_mega_bytes(s_bv)
-        + size_in_mega_bytes(i_iv)
+
+        //+ size_in_mega_bytes(i_iv)
```

Comment:
These can be cleaned up too. Also, not that this is your code, but isn't s_bv still part of the XG? Seems to me that it shouldn't be commented out.